gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java

```diff
@@ -97,6 +109,19 @@ public final File getExecutableDir() {
         return productDependencies;
     }
 
+    @Input
+    @Optional
+    public final MapProperty<String, Serializable> getConjureExtensions() {
```

Comment:
@carterkozak flagged that this could break when the task is invoked from command line and e.g. the command line has a max limit for input args if the extension data exceeds the limit.

But if it's working for you project we can handle such issues when they arise -- just something to keep in mind.

Reply:
You mean when passing to the actual generator?  I just wrote a quick test that passes an extension of ~9M and it worked on my local mac, which I presume means it will work on linux.  There might be an issue on Windows, but I think that would be a rare case and, as you say, can address if it comes up.

Reply:
Thanks for adding the test! Yes, when passing it to the generator executable here:

https://github.com/palantir/gradle-conjure/blob/cbaa18cd6056459b365d369d6cb2d0d49067ad8a/gradle-conjure/src/main/java/com/palantir/gradle/conjure/CompileIrTask.java#L132-L133

Yep, might only be an issue in specific environments (windows maybe?). Should be good for now, we can think about this more if it becomes an actual problem (e.g. pass the extension by file to the executable).